src/components/Header/Header.scss

```diff
@@ -1,9 +1,150 @@
 @import '~style/variables';
 
-.wrapper {
-  border: 1px solid blue;
+header {
+  display: flex;
+  flex-direction: column;
+  background-color: $color-primary--500;
+  color: $color-white;
+
+  > * {
+    @include for-desktop-up {
+      padding-left: 30px;
+      padding-right: 30px;
+      max-height: 176px;
+    }
+  }
+
+  a {
+    color: white;
+    text-decoration: underline;
+    text-decoration-color: $color-highlight;
+  }
+}
+
+.govContainer {
+  display: flex;
+  height: 48px;
+  padding-left: 14px;
+  flex-flow: column wrap;
+  align-items: flex-start;
+  align-content: flex-start;
+  justify-content: center;
+
+  @include for-desktop-up {
+    padding-left: 30px;
+    height: initial;
+    align-items: baseline;
+    flex-flow: row nowrap;
+    justify-content: flex-start;
+    background-color: $color-primary--400;
+
+    > * {
+      margin-right: 8px;
+    }
+  }
+}
+
+.compactTitle,
+.compactSubtitle {
+  @include for-desktop-up {
+    display: none;
+  }
+}
+
+.govAu,
+.govWebsite {
+  display: none;
+
+  @include for-desktop-up {
+    display: initial;
+  }
+}
+
+.govAu {
+  font-size: 14px;
+  font-weight: 500;
+}
+
+.govWebsite {
+  font-size: 12px;
```

Comment:
I *think* font-size should go from everywhere. The dream is to have all typography defined in one place. This works well if there's a finite number of styles, and I think Nick is keen to have that. But if every bit of text is different maybe that's all a waste of time.

Thoughts?

Reply:
I can't see a global style that corresponds to this. I think: best to set global styles in typography file, but not clutter it with snowflakes.